runtime/doc/windows.txt

```diff
@@ -127,6 +127,8 @@ You can change the contents of the status line with the 'statusline' option.
 This option can be local to the window, so that you can have a different
 status line in each window.
 
+Note: |floating-windows| 'statusline' is not affected by 'laststatus'.
```

Comment:
Does this mean floating windows still show statusline even with laststatus 0 or 3?